### ISSUE TITLE
onnx/utils._create_jit_graph() got problem

### ISSUE THREAD

**Issue by yingjie-jiang** [onnx/onnx#4026]
use export.py convert yolov5s.pt(from [ultralytics/yolov5](https://github.com/ultralytics/yolov5)) to onnx failed, but don't get any errors.
When I debug, the code list(graph.inputs()) in function onnx/utils._create_jit_graph() runs unresponsive.

def _create_jit_graph(model, args, _retain_param_name):
    torch_out = None
    params: Union[List, Tuple]

    if isinstance(model, torch.jit.ScriptModule):
        try:
            graph = model.forward.graph
            torch._C._jit_pass_onnx_function_substitution(graph)
            freezed_m = torch._C._freeze_module(model._c, preserveParameters=True)
            module, params = torch._C._jit_onnx_list_model_parameters(freezed_m)
            method_graph = module._get_method('forward').graph
            args_params = tuple(args) + tuple(params)
            param_count_list = _get_param_count_list(method_graph, args_params)
            in_vars, _ = torch.jit._flatten(args_params)
            graph = _propagate_and_assign_input_shapes(
                method_graph, tuple(in_vars), param_count_list, False, False)
        except AttributeError as e:
            raise RuntimeError('\'forward\' method must be a script method') from e
        return graph, params, torch_out, module
    elif isinstance(model, torch.jit.ScriptFunction):
        params = ()
        in_vars, in_desc = torch.jit._flatten(tuple(args))
        graph = model.graph
        torch._C._jit_pass_onnx_function_substitution(graph)
        param_count_list = _get_param_count_list(graph, args)
        graph = _propagate_and_assign_input_shapes(
            graph, tuple(in_vars), param_count_list, False, False)
        return graph, params, torch_out, None
    else:

        graph, torch_out = _trace_and_get_graph_from_model(model, args)

        state_dict = _unique_state_dict(model)

        params = list(state_dict.values())

        if _retain_param_name:
            try:
                print("start==============",)
                graph_inputs = list(graph.inputs())
            except Exception as result:
                print(result)
            print("start1==============", graph_inputs)
            user_input_num = len(graph_inputs) - len(state_dict)
            param_names = list(state_dict.keys())

            for i, inp in enumerate(graph_inputs):
                if i >= user_input_num:
                    inp.setDebugName(param_names[i - user_input_num])

        torch._C._jit_pass_onnx_function_substitution(graph)
        return graph, params, torch_out, None

export: data=data\bottle.yaml, weights=weights/yolov5s.pt , imgsz=[416], batch_size=1, device=cpu, half=False, inplace=False, train=False, optimize=False, int8=False, dynamic=False, simplify=False, opset=12, verbose=False, workspace=4, nms=False, agnostic_nms=False, topk_per_class=100, topk_all=100, iou_thres=0.45, conf_thres=0.25, include=['onnx']
YOLOv5  2022-2-15 torch 1.9.0+cu111 CPU

Fusing layers... 
Model Summary: 213 layers, 7225885 parameters, 0 gradients

PyTorch: starting from weights\yolov5s.pt  with output shape (1, 10647, 85) (14.7 MB)

ONNX: starting export with onnx 1.10.2...
start==============

Process finished with exit code -1073741819 (0xC0000005)

**Comment by yingjie-jiang** [comment]
import onnx must front of import torch
